src/App.js

```diff
@@ -1,20 +1,26 @@
 import "./App.css";
-import React from "react";
+import React, { useState } from "react";
 import Header from "./Components/Header";
 import Nav from "./Components/Nav";
 import Reviews from "./Components/Reviews";
 import SingleReview from "./Components/SingleReview";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
+import SingleCategory from "./Components/SingleCategory";
 
 function App() {
+  const [currentUser, setCurrentUser] = useState("jessjelly");
   return (
     <BrowserRouter>
       <div className="App">
         <Header />
         <Nav />
         <Routes>
           <Route path="/" element={<Reviews />} />
-          <Route path="/reviews/:reviewID/*" element={<SingleReview />} />
+          <Route path="/reviews/:category_slug" element={<SingleCategory />} />
+          <Route
+            path="/reviews/:reviewID/*"
+            element={<SingleReview currentUser={currentUser} />}
```

Comment:
This works, however, the component is exactly the same as the <Reviews/> component, except the api request is slightly different. The code isn't that dry. 
It would be better to re-use the <Reviews/> component